.github/workflows/build-sw-container.yml

```diff
@@ -38,8 +38,13 @@ jobs:
         TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
       run: |
         docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
-        tag_list=$docker_repo:$REPO,$docker_repo:$REPO-$TIMESTAMP
-        echo "::set-output name=taglist::$tag_list"
+        tag_list=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          for image_tag in "$REPO" "$REPO-$TIMESTAMP"; do
+            tag_list+=("$registry/$docker_repo":"$image_tag")
+          done
+        done
+        echo "::set-output name=taglist::${tag_list[*]}"
```

Comment:
@brianhlin just fyi I think your `${tag_list[*]}` here will expand with spaces rather than comma-separated (unless you are setting `IFS=","`).

Not sure if that's what you want, as it looks like the previous version has them comma-separated.

Reply:
Hrm yeah, I forget if that was on purpose or what but it still seems to work 🤷🏽 : https://github.com/opensciencegrid/docker-xrootd-standalone/runs/3212639548?check_suite_focus=true#step:7:13